seed_stage_based_messaging/settings.py

```diff
@@ -144,6 +147,15 @@
     "dsn": STAGE_BASED_MESSAGING_SENTRY_DSN
 }
 
+CACHES = {
+    "default": env.cache(default="locmemcache://"),
+    "locmem": {
+        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+        "TIMEOUT": 60,
```

Comment:
I think this timeout is in seconds, so this will need to be `60 * 60` for 60 minutes